stripe/request_metrics.py

```diff
@@ -0,0 +1,10 @@
+class RequestMetrics(object):
```

Comment:
Could you add `from __future__ import absolute_import, division, print_function` at the beginning of the file? Even though that line won't do anything here, it's present in all source files to prevent Python 2/3 compatibility issues.